ddtrace/contrib/botocore/patch.py

```diff
@@ -20,6 +23,41 @@
 ARGS_NAME = ('action', 'params', 'path', 'verb')
 TRACED_ARGS = ['params', 'path', 'verb']
 
+propagator = HTTPPropagator()
+
+
+def modify_client_context(client_context_base64, trace_headers):
```

Comment:
Hmm do you know if it's the job of the user to encode the context or is it done in botocore itself? If there's an earlier point where we can intercept the context before it's encoded that would be great. 

I'd strongly prefer, if possible, to not have to decode and re-encode for performance and maintainability reasons.

Reply:
Unfortunately, it's the job of the user to encode the context so intercepting this earlier is not an option here 👎

Reply:
ah darn! I wouldn't mind having some error handling here in case the encoding isn't what's expected.

A simple `try..except Exception` with a warning log would be great

Reply:
Added!